extension/scripts/features/shop-filters/ttShopFilters.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+(async () => {
+	if (!getPageStatus().access) return;
+
+	featureManager.registerFeature(
+		"Shop Filters",
+		"shops",
+		() => settings.pages.shops.filters,
+		null,
+		addFilters,
+		removeFilters,
+		{
+			storage: ["settings.pages.shops.filters"],
+		},
+		() => {
+			if (!hasAPIData()) return "No API access.";
+		}
+	);
+
+	const localFilters = {};
+	async function addFilters() {
+		const title = await requireElement(".buy-items-wrap > [role*='heading']");
+
+		const shopFilters = createCheckboxList({
+			items: [
+				{ id: "hideLoss", description: "Hide items with loss" },
+				{ id: "hideUnder100", description: "Hide items under 100 stock" },
+			],
+			orientation: "row",
+		});
+		shopFilters.setSelections([ ...filters.shops.hideLoss ? ["hideLoss"] : [], ...filters.shops.hideUnder100 ? ["hideUnder100"] : [] ]);
+
+		shopFilters.onSelectionChange(filtering);
+		localFilters.filters = { getSelections: shopFilters.getSelections };
+
+		filtering();
+
+		title.appendChild(
+			document.newElement({
+				type: "div",
+				class: "tt-shop-filters tt-theme",
+				children: [ shopFilters.element ],
+			})
+		);
+	}
+
+	async function filtering() {
+		await requireElement(".item-desc");
+
+		const checkboxes = localFilters.filters.getSelections();
+		const hideLoss = checkboxes.includes("hideLoss");
+		const hideUnder100 = checkboxes.includes("hideUnder100");
+
+		await ttStorage.change({ filters: { shops: { hideLoss, hideUnder100 } } });
+
+		for (const li of document.findAll(".buy-items-wrap .items-list > li:not(.empty, .clear)")) {
```

Comment:
What what, does that work? Didn't know that, nice one.

Reply:
Took some time to find it but it worked.